org-formation/075-security-hub/security-hub-suppress-lambda.yaml

```diff
@@ -0,0 +1,119 @@
+---
```

Comment:
did you create this lambda?  if not could you please add a comment telling us where you got it from?

Reply:
Added source in desc.